newIDE/app/src/AssetStore/index.js

```diff
@@ -792,7 +798,8 @@ export const AssetStore = React.forwardRef<Props, AssetStoreInterface>(
               hideGameTemplates={hideGameTemplates}
               hideDetails={!!assetSwappedObject}
             />
-          ) : openedAssetShortHeader ? (
+          ) : // Do not show the asset details if we're swapping an asset.
+          openedAssetShortHeader && !assetSwappedObject ? (
```

Comment:
that logic is both for asset swapping in the objects list & asset swapping in the quick customization.
I'm wondering if we should differentiate both.
-> quick custom, keep it simple
-> objects list, give all the search options, including seeing the details page

Reply:
Let's keep it simple for now but yes we probably want to give the full details in the objects list (and even later an option to import from a file)

Reply:
updating this with a minimalUI param. Otherwise, we're losing features